### PR TITLE
[Method References] LPS-160434 Replace method setPortlet with field _portlet in class extends BasePanelApp

### PR DESCRIPTION
**Pull request by W2jx**
Hi @tinatian @dantewang , I have replaced method setPortlet with field _portlet and removed method setUserNotificationEventLocalService in this pr

Detail: https://github.com/liferay-core-infra/liferay-portal/pull/1416

### CODE REVIEW

**Comment by W2jx** [comment]
Hi @catarineaguiar , Could you please help me check the test results when they finish ? Thanks!

**Comment by dantewang** [comment]
ci:test:sf

File: modules/apps/application-list/application-list-api/src/main/java/com/liferay/application/list/BasePanelApp.java [hunk 1]
```diff
@@ -99,15 +99,21 @@ public int getNotificationsCount(User user) {
...
 			getUserNotificationEventsCount(
-				user.getUserId(), _portlet.getPortletId(),
+				user.getUserId(), portlet.getPortletId(),
 				UserNotificationDeliveryConstants.TYPE_WEBSITE, true, false);
 	}
 
 	@Override
 	public Portlet getPortlet() {
-		return _portlet;
+		if (_portletLocalService != null) {
```
Comment (tinatian) [inline modules/apps/application-list/application-list-api/src/main/java/com/liferay/application/list/BasePanelApp.java:101]: @W2jx Why this is needed? All sub classes should implement this method rather that have a implementation in base method.

Comment (W2jx) [reply modules/apps/application-list/application-list-api/src/main/java/com/liferay/application/list/BasePanelApp.java:101]: I will fix them.

**Comment by dantewang** [comment]
@W2jx As we talked, please resend.

**Pull request closed without merging**